scripts/build.js

```diff
@@ -102,6 +103,7 @@ let spec = await dereferenceDocument(doc);
 
 spec.components = {};
 
+// Merge instances of `allOf` in methods.
 function recursiveMerge(schema) {
   schema = merger(schema);
```

Comment:
@lightclient Unrelated, but it looks like `merger` is [recursive](https://github.com/mokkabonna/json-schema-merge-allof#options) by default. I checked against a simple test case involving `oneOf`, and it seems to work. Does this function address some corner case?

Reply:
No diff in generated spec either 🤔

Reply:
It looks like you're right. I'm not sure if addressed an edge case at one point, but for it appears to be superfluous. Do you mind creating a separate PR to remove the function and use only merger?